src/components/metadataEditor/metadataEditor.js

```diff
@@ -1088,6 +1128,7 @@ define(['itemHelper', 'dom', 'layoutManager', 'dialogHelper', 'datetime', 'loadi
                     elem.innerHTML = globalize.translateDocument(template, 'core');
 
                     elem.querySelector('.formDialogFooter').classList.remove('formDialogFooter');
+                    elem.querySelector('.btnClose').classList.add('hide');
```

Comment:
Isn't this the same as `btnCancel`?

Reply:
The purpose of that other selector is to have the cancel button outside of some cases but hide the top-right X button.